apps/codeforafrica/src/lib/api.netlify-cms/getTeam.js

```diff
@@ -0,0 +1,15 @@
+import { join } from "path";
+
+import getCollectionData from "./getCollectionData";
+
+const teamDir = join(process.cwd(), "content/team");
+
+export default function getTeam(fields) {
+  const team = getCollectionData(teamDir, fields);
+  team.reduce((acc, curr) => {
+    const member = curr;
+    member.href = `/about/members/${member.slug}`;
+    return acc;
+  }, []);
```

Comment:
Is team an object or why are we using `reduce` instead of `map`?

Reply:
fixed that.